Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs

```diff
@@ -44,10 +44,11 @@ public sealed class UdpServerEntryPoint : IServerEntryPoint
         }
 
         /// <inheritdoc />
-        public async Task RunAsync()
+        public Task RunAsync()
```

Comment:
hmm, it seems it needs renaming

Reply:
Why?

Reply:
because it's no longer an `async` function?

Reply:
Still returns a `Task`, so it isn't any different from the callers perspective